packages/arb-token-bridge-ui/src/components/common/NetworkImage.tsx

```diff
@@ -3,8 +3,7 @@ import { getNetworkName, isNetwork } from '../../util/networks'
 import { getBridgeUiConfigForChain } from '../../util/bridgeUiConfig'
 
 export const NetworkImage = ({
-  chainId,
-  theme = 'light'
+  chainId
```

Comment:
We don't need the theme anymore. It was only used to show white or black custom orbit chain logo.

The current custom orbit chain logo is blue-ish, so it fits any scenario.